src/test/java/org/openhab/binding/jrule/internal/triggers/itemchange/JRuleGroupItemChangeTest.java

```diff
@@ -49,74 +50,110 @@ public void initTestClass() throws ItemNotFoundException {
                 .then((Answer<Item>) invocationOnMock -> new StringItem(invocationOnMock.getArgument(0)));
     }
 
-    @Test
-    public void testItemChange_selfGroup() {
-        JRuleGroupItemChangeRules rule = initRule(JRuleGroupItemChangeRules.class);
-        // Only last event should trigger rule method
-        fireEvents(List.of(itemChangeEvent(JRuleGroupItemChangeRules.GROUP_ITEM, "2", "1")));
-        verify(rule, times(0)).groupItemChange(Mockito.any(JRuleEvent.class));
-    }
-
-    @Test
-    public void testItemChange_no_from_to() throws ItemNotFoundException {
-        Mockito.when(itemRegistry.getItem(MEMBER_ITEM)).then((Answer<Item>) invocationOnMock -> {
-            StringItem stringItem = Mockito.mock(StringItem.class);
-            Mockito.when(stringItem.getName()).thenReturn(invocationOnMock.getArgument(0));
-            Mockito.when(stringItem.getGroupNames()).thenReturn(List.of(JRuleGroupItemChangeRules.GROUP_ITEM));
-            return stringItem;
-        });
-
-        JRuleGroupItemChangeRules rule = initRule(JRuleGroupItemChangeRules.class);
-        // Only last event should trigger rule method
-        fireEvents(List.of(itemChangeEvent(OTHER_ITEM, "2", "1"), itemChangeEvent(MEMBER_ITEM, "2", "1")));
-        verify(rule, times(1)).groupItemChange(Mockito.any(JRuleEvent.class));
-    }
-
-    @Test
-    public void testItemChange_from() throws ItemNotFoundException {
-        Mockito.when(itemRegistry.getItem(MEMBER_ITEM)).then((Answer<Item>) invocationOnMock -> {
-            StringItem stringItem = Mockito.mock(StringItem.class);
-            Mockito.when(stringItem.getName()).thenReturn(invocationOnMock.getArgument(0));
-            Mockito.when(stringItem.getGroupNames()).thenReturn(List.of(JRuleGroupItemChangeRules.GROUP_ITEM_FROM));
-            return stringItem;
-        });
-
-        JRuleGroupItemChangeRules rule = initRule(JRuleGroupItemChangeRules.class);
-        // Only last event should trigger rule method
-        fireEvents(List.of(itemChangeEvent(MEMBER_ITEM, "2", "1"), itemChangeEvent(MEMBER_ITEM, "1", "2")));
-        verify(rule, times(1)).groupItemChangeFrom(Mockito.any(JRuleEvent.class));
-    }
+//    @Test
```

Comment:
Same here - obsolete or temporarily commented out while waiting for another PR to merge?

Reply:
uncommented